sorc/grid_tools.fd/regional_esg_grid.fd/psym2.f90

```diff
@@ -35,18 +36,17 @@ module psym2
 
 contains
 
-!=============================================================================
+!> Get the orthogonal eigenvectors, vv, and diagonal matrix of eigenvalues, oo,
+!! of the symmetric 2*2 matrix, em.
+!! @author R. J. Purser
```

Comment:
Move author tag to bottom in all cases.

Reply:
> Move author tag to bottom in all cases.

Would you please explain where should the @author tag located. I do not understand your comment - Move author tag to bottom in all cases.
You may use grid_tools.fd/regional_esg_grid.fd/psym2.f90 as an example.